scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -38,9 +38,8 @@ test-dependency-rules:
 test-rust-features:
   stage:                           check
   extends:
-    - .kubernetes-env
+    - .docker-env
```

Comment:
If there is no speed improvement I'd keep it on k8s runners

Reply:
Okay. It seems bound on the download speed, not CPU. Seems to slow down the second stage of the CI by about 30s, so if that is a problem we can maybe move it into the next stage.

Reply:
The pipeline takes about 20-25 minutes, so I don't think it's a big problem